core/lib/slider.ts

```diff
@@ -297,13 +297,13 @@ export function createSlider(config?: PropsConfig<SliderProps>): SliderWidget {
 	const sliderDomRectOffset$ = computed(() => (vertical$() ? sliderDomRect$().top : sliderDomRect$().left));
 	const sliderDomRectSize$ = computed(() => (vertical$() ? sliderDomRect$().height : sliderDomRect$().width));
 	const sortedValues$ = computed(() => [...values$()].sort((a, b) => a - b));
-	const sortedHandles$ = computed(() =>
-		values$()
+	const sortedHandles$ = computed(() => {
+		return values$()
```

Comment:
Here I prefer the previous syntax as it is a single expression.

Reply:
Oh yes, it's just because it's going to be change with the next PR I'm working on, on top of this one...